src/functionalTest/resources/scenarios/RIA-2239-judge-end-appeal.json

```diff
@@ -0,0 +1,101 @@
+{
+  "description": "RIA-2239 end appeal",
+  "enabled": "{$featureFlag.docmosisEnabled}",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "JudgeX",
```

Comment:
Should also be a 'Judge' credential (not a JudgeX), to be in line with the other roles.